app/models/concerns/twitter_user/utils.rb

```diff
@@ -0,0 +1,47 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Utils
```

Comment:
__[rubocop]__
* Missing top-level module documentation comment.
* Use nested module/class definitions instead of compact style.